MeetingScheduler/Logger/DBLogger.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    public class DBLogger : LogBase
+    {
+        public override void Log(string message)
+        {
+            lock (lockObj)
+            {
```

Comment:
Sto se tice db logger-a ne bih to implementirao, ali razmisli koje su prednosti i mane takvog upisa i izlozi ih ovde.

Reply:
Ako bi postojalo vise masina koje koriste moju aplikaciju, one bi u deljenu bazu podataka mogle da loguju. Pa bi vise ljudi imalo pristup logovima recimo. Nemoguce bi bilo da se logovi obrisu sa masine na kojoj pokrecemo aplikaciju isto.

Sto se tice mana, mal osam citao, a kao glavne se pojavljuju 
- preopterecenost baze operacijama upisivanja, sto bi usporilo generalni rad sa bazom
- problemi sa lokovanjem pristupa bazi, prilikom upisivanja sa razlicitih masina

Reply:
Uglavnom je to to, plus ako dodas i cryptovanje koje cemo i naovom projektu raditi onda to moze da bude bas komplikovano.